src/Fmt_odoc.ml

```diff
@@ -12,6 +12,12 @@
 open Octavius.Types
 open Fmt
 
+let str s =
+  String.substr_replace_all s ~pattern:"@" ~with_:"\\@"
+  |> String.substr_replace_all ~pattern:"{" ~with_:"\\{"
+  |> String.substr_replace_all ~pattern:"}" ~with_:"\\}"
```

Comment:
Maybe it makes sense to also add `[` and `]`

Reply:
It could very well. Might address one of my comments on your infer PR?

Reply:
Also, it might make sense to use `Base.String.Escaping` for this to avoid a potential performance problem.